### ISSUE TITLE
Update manger and scheduler conatiner images

### ISSUE THREAD

**Issue by kelson42** [offspot/imager-service#422]
… they are very old and should be refreshed. They also be renamed.

**Comment by rgaudin** [comment]
> … they are very old and should be refreshed. 

No

> They also be renamed.

Indeed

**Comment by end-9214** [comment]
i want to work on this, can you assign this to me

**Comment by rgaudin** [comment]
No, this is a tiny task with deployment consequences ; we'll do it ourselves
